Lib/test/test_tarfile.py

```diff
@@ -3801,6 +3801,77 @@ def __exit__(self, *exc_info):
         self.assertEqual(custom_filter.num_files_processed, 2)
         self.assertEqual(custom_filter.done, True)
 
+    def test_errorlevel(self):
+        def extracterror_filter(tarinfo, path):
+            raise tarfile.ExtractError('failed with ExtractError')
+        def tarerror_filter(tarinfo, path):
+            raise tarfile.ExtractError('failed with base TarError')
+        def filtererror_filter(tarinfo, path):
+            raise tarfile.FilterError('failed with FilterError')
+        def oserror_filter(tarinfo, path):
+            raise OSError('failed with OSError')
+        def valueerror_filter(tarinfo, path):
+            raise ValueError('failed with ValueError')
+
+        with ArchiveMaker() as arc:
+            arc.add('file')
+
+        # If errorlevel is 0, errors are ignored when using TarFile.extract().
+
+        with self.check_context(arc.open(errorlevel=0), tarerror_filter):
+            self.expect_file('file')
+
+        with self.check_context(arc.open(errorlevel=0), extracterror_filter):
+            self.expect_file('file')
+
+        with self.check_context(arc.open(errorlevel=0), filtererror_filter):
+            self.expect_file('file')
+
+        with self.check_context(arc.open(errorlevel=0), oserror_filter):
+            self.expect_file('file')
+
+        with self.check_context(arc.open(errorlevel=0), valueerror_filter):
+            self.expect_exception(ValueError)
```

Comment:
This test is under a "errors are ignored" comment and it expects an error not to be ignored. Could you add a comment above? E.g. "Other errors are never ignored". Thanks.